src/utils/testHelpers.ts

```diff
@@ -33,19 +42,21 @@ export const createFakeHttpResponse = (
   content: string | object,
   request: HttpRequestOptions = {}
 ): HttpResponse => {
-  const stringifiedContent = typeof (content) === 'object' ? JSON.stringify(content) : content
+  const stringifiedContent =
+    typeof content === 'object' ? JSON.stringify(content) : content
   return {
     status,
     content: stringifiedContent,
     headers: {},
     getHeader: (key: string) => undefined,
     throwForStatus: () => ({}),
-    request
+    skipThrowForStatus: false,
```

Comment:
Should we apply this for all tests? I believe that default in recent versions of ZapierPlatformCore is `true`, isn't it?